### DISCUSSION TITLE
List of app

### DISCUSSION THREAD

**Discussion by vercorsio** [timdorr/tesla-api#298] [General]
Hey Tim

You are a star. (I mean it)
I follow your work for years and I'm using it to build app that connects to tesla servers. As many of us. A big thank for this.

Since Tesla as changed its authentication server, a lot of developers are impacted and visit your GitHub repo to update and keep their app up and running. I realise that it makes a huge community.

I was wondering to know if you could add to your https://tesla-api.timdorr.com/ a page that lists all the products/app/softwares that use or are inspired by your great work.
Just declarative, moderated by you or one of your contributors

What do you think ?

Keep safe and drive safely !

Nicolas

**Comment by timdorr** [comment]
Yeah, if someone wants to put together a docs page with a link to all of that, that would be great. Otherwise, I've found either Discussion or Issue posts to work. Either way would be good!

**Reply by vercorsio** [reply]
Great. Let's start here and use this discussion to let developers presenting their use of this API.
I propose:  
 - **application name** : what's the name of your "software"/"product"/"app"
 - **short description** : "few word to describe why, where, what it does
 - **creation date** : when first version has been published
 - **country** : where are you from
 - **url to web site** : any web site / market link / Git repo
 - **email** : how to contact you
 - **platform(s)** : iOS / Android / Desktop / Mobile / Home Automation system / ...
 - **language/framework** : Ruby, Java, PHP, JS, ..., Vue.js, Angular, Flutter, ...) . 

Would you like to start Tim ?
